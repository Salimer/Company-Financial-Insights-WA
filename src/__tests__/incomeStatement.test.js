import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import IncomeStatement from '../components/incomeStatement';

test('renders the income statement correctly', () => {
  const company = [
    // Mock company data
    {
      date: '2023-01-01',
      reportedCurrency: 'USD',
      revenue: 1000000,
      costOfRevenue: 800000,
      grossProfit: 200000,
    },
  ];

  const { container } = render(
  <MemoryRouter>
    <IncomeStatement company={company} />
  </MemoryRouter>
  );
  expect(container).toMatchSnapshot();
});
