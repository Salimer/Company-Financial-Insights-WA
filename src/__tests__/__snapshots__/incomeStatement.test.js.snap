// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the income statement correctly 1`] = `
<div>
  <ul
    class="flex flex-col h-screen justify-between"
  />
</div>
`;
